uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/util/commands/CreateRepository.java

```diff
@@ -104,6 +139,17 @@ public CreateRepository(final File repoDir,
         }
     }
 
+    private void setupSSLVerify(Repository repo) throws java.io.IOException {
+        StoredConfig config = repo.getConfig();
+        if (sslVerify) {
+            config.setBoolean("http", null, "sslVerify", true);
+            config.save();
+        } else {
+            config.setBoolean("http", null, "sslVerify", false);
```

Comment:
@domhanak

Reply:
Nitpick The same could have been accomplished without if-else:
```java
config.setBoolean("http", null, "sslVerify", sslVerify);
config.save();
```

But don't let that block merging this PR.